.github/workflows/build.yml

```diff
@@ -0,0 +1,102 @@
+name: Build and test
+
+on: [push]
+
+jobs:
+  build:
+    name: Build and test for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-2016, macOS-latest]
+    steps:
+      - name: Enable symlinks
+        run: git config --global core.symlinks true
+
+      - name: Checkout the repository
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+          submodules: true
+
+      - name: install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: install rust
+        uses: hecrj/setup-rust-action@v1
+        with:
+          rust-version: "1.37.0"
+
+      - name: install clippy and rustfmt
+        run: |
+          rustup component add clippy
+          rustup component add rustfmt
+
+      - name: install python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "2.7.16"
+          architecture: x64
+
+      - name: Setup python package dir
+        if: startsWith(matrix.os, 'windows')
+        run: setx PYTHONPATH "third_party\python_packages" /m
```

Comment:
I don't think `setx` is the command to use here; it's similar to adding something to `~/.profile` in that it is a permanent change but does not take effect immediately.
The proper invocation is `set PYTHONPATH=third_party\python_packages`.

Reply:
It should probably be echo ::set-env PYTHONPATH=third_party\python_packages. This is inter process persistent just like setx. set won't work because it sets the variable for one process only.